src/main/webapp/package.json

```diff
@@ -6,6 +6,7 @@
     "start": "ng serve",
     "build:dev": "mkdir -p ../../../target/growpod-1/WEB-INF && cp -R src/WEB-INF ../../../target/growpod-1/ && ng build --configuration=dev",
     "build": "ng build",
+    "build:prod": "ng build --configuration=production",
```

Comment:
afaik, we only need a `build:dev` and a `build:prod` right? no need for three

Reply:
Will consolidate.